src/aspire/basis/fspca.py

```diff
@@ -31,15 +31,19 @@ class FSPCABasis(SteerableBasis2D):
 
     """
 
-    def __init__(self, src, basis=None, noise_var=None, components=400, batch_size=512):
+    def __init__(
+        self, src, basis=None, noise_var=None, components=None, batch_size=512
+    ):
         """
 
         :param src: Source instance
         :param basis: Optional Fourier Bessel Basis (usually FFBBasis2D)
+        :param components: Number of components (compression).
+        Defaults None implies basis.count, therefor no compression.
         :param noise_var: None estimates noise (default).
-        :param batch_size: Batch size for computing basis coefficients.
         0 forces "clean" treatment (no weighting).
         Other values assigned to noise_var.
```

Comment:
I know you didn't touch this line but maybe the blurb for `noise_var` could be reworded? Not sure what the second part is trying to say